hacking/test-module

```diff
@@ -201,6 +201,8 @@ def runtest(modfile, argspath, modname, module_style, interpreters):
             invoke = "%s " % interpreters['ansible_python_interpreter']
         else:
             invoke = ""
+    else:
```

Comment:
Probably easier to set `invoke = ""` at the top of the function and avoid two else clauses.

`invoke` could also be better named as `interpreter` or something.

Sorry about this bug btw, I must have missed this case while trying to make test-module work under virtualenvs / OS X brew python